reftests/local.ron

```diff
@@ -1,10 +1,30 @@
 {
```

Comment:
Not a big deal, but does ron require tab spacing? Spaces would be a bit cleaner for these diffs if we could reformat the reftests at some point